maintenance_security/__manifest__.py

```diff
@@ -9,7 +9,7 @@
     "website": "https://github.com/OCA/maintenance",
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "license": "AGPL-3",
-    "depends": ["maintenance"],
+    "depends": ["hr", "maintenance"],
```

Comment:
```suggestion
    "depends": ["maintenance"],
```
IMO `hr` dependency is not necessary.

Reply:
True, the requiring entry was split to module hr_maintenance_security.